docs/architecture/adr-008-dCERT-group.md

```diff
@@ -0,0 +1,149 @@
+# ADR 008: Decentralized Computer Emergency Response Team (dCERT) Group
+
+## Changelog
+
+- 2019 Jul 31: Initial Draft
+
+## Context
+
+In order to reduce the number of parties involved with handling sensitive
+information in an emergency scenario, we propose the creation of a
+specialization group named The Decentralized Computer Emergency Response Team
+(dCERT).  Initially this group's role is intended to serve as coordinators
+between various actors within a blockchain community such as validators,
+bug-hunters, developers.  During a time of crisis, the dCERT group would
+aggregate and relay input from a variety of stakeholders to the developers who
+are actively devising a patch to the software, this way sensitive information
+does not need to be publicly disclosed while some input from the community can
+still be gained. 
+
+Additionally, a special privilege is proposed for the dCERT group:
+the capacity to "circuit-break" (aka. temporarily disable)  a particular
```

Comment:
I believe we talked about initially having the implementation _not_ include this functionality but which can later be easily added. I think we should make note of that here 👍

Reply:
Added the text

> Note that this privilege should be enabled/disabled globally with a governance
parameter such that this privilege could start disabled and later be enabled
through a parameter change proposal, once a dCERT group has been established.